usr.sbin/daemon/daemon.c

```diff
@@ -431,28 +424,8 @@ main(int argc, char *argv[])
 	state.pipe_fd[1] = -1;
 
 	setproctitle("%s[%d]", state.title, (int)pid);
-	/*
```

Comment:
This doesn't seem to be fundamentally changed by this commit, and AFAICT it's still true; why remove the comments?

Reply:
I consolidated scattered comments that talk about signal handling nuances in one place, closer to the beginning of the main()